priv/templates/boot.eex

```diff
@@ -2,6 +2,7 @@
 
 set -e
 
+command -v erl >/dev/null 2>&1 || { echo "erlang runtime not found. If erlang is installed check your $PATH. aborting.";  exit 1; }
```

Comment:
Hmm, the only place `erl` is pulled from the path is in `_find_erts_dir` on line 115, I think this check should be there instead, since we don't need `erl` to be on the path if we're bundling ERTS.

Also, this command will start erl, but not stop it, leaving a dead VM instance laying around. The command should be `erl -noshell -eval "halt()."`, which will start the VM and immediately stop it without attaching a shell.